Flutter/circular-charts/chart-types/radial-bar-chart.md

```diff
@@ -155,9 +162,56 @@ The [`cornerStyle`](https://pub.dev/documentation/syncfusion_flutter_charts/late
     }
 
 {% endhighlight %}
+{% endtabs %}
 
 ![Radial bar corner style](circular-chart-types-images/radialbar_roundCorner.jpg)
 
+## Track opacity
```

Comment:
Remove the heading and content and use the below content. Also move this after `Rounded corners` section and need to remove the old content as in below screenshot

![image](https://user-images.githubusercontent.com/19346074/160795429-4e8208b9-5a78-4d73-b0ca-0ddeb654a9ac.png)


## Track customization

The radial bar series renders above the track area. The following properties are used to customize the appearance.

* [`trackColor`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackColor.html) - changes the color of the track area.
* [`trackBorderColor`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackBorderColor.html) - changes the stroke color of the track area.
* [`trackBorderWidth`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackBorderWidth.html) - changes the stroke width of the track area.
* [`trackOpacity`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/trackOpacity.html) - specifies the opacity of the track area.
* [`useSeriesColor`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/RadialBarSeries/useSeriesColor.html) - uses the point color for filling the track area.

Reply:
added the above content